### PR TITLE
Added missing gpu message instead of panicking on 'None'.

### LINKED ISSUE #100: Provide a helpful error message when render backends cannot be initialized
As pointed out in #96, if someone doesn't have their graphics drivers/libs set up correctly an ugly and unhelpful panic occurs. We should provide a helpful message.

### PR DESCRIPTION
**Pull request by MGlolenstine**
I've added a better message to let people know what's wrong instead of just panicking on an `unwrap() `.

This one should close #100 .

### CODE REVIEW

**Comment by cart** [comment]
Looks good to me. Thanks!

File: crates/bevy_wgpu/src/wgpu_renderer.rs [hunk 1]
```diff
@@ -19,13 +19,15 @@ pub struct WgpuRenderer {
 impl WgpuRenderer {
     pub async fn new() -> Self {
         let instance = wgpu::Instance::new(wgpu::BackendBit::PRIMARY);
-        let adapter = instance
+        let adapter = if let Some(a) = instance
```
Comment (cart) [inline crates/bevy_wgpu/src/wgpu_renderer.rs:22]: I think we can make this a little more terse like so:
```rust
instance
            .request_adapter(&wgpu::RequestAdapterOptions {
            .request_adapter(&wgpu::RequestAdapterOptions {
                power_preference: wgpu::PowerPreference::Default,
                power_preference: wgpu::PowerPreference::Default,
                compatible_surface: None,
                compatible_surface: None,
            })
            }).await.expect("Unable to find a GPU! Make sure you have installed required drivers!")
```

Comment (MGlolenstine) [reply crates/bevy_wgpu/src/wgpu_renderer.rs:22]: I'll take a look at it tomorrow/later today, as it's 0:24 and I've got work in about 5 hours 😅
Thanks!

Comment (cart) [reply crates/bevy_wgpu/src/wgpu_renderer.rs:22]: Thank you :)

**Comment by cart** [comment]
Alrighty lets merge this!

**Pull request merged**